components/AppCatalog.vue

```diff
@@ -26,6 +26,8 @@ const flatTracks = computed<JamfluencerApi.CatalogTrack[]>(() => {
   ];
 });
 
+const artistList = (track) => track.artists.map((a) => '<a href="'+a.link+'">'+a.name+'</a>').join(', ');
```

Comment:
This looks like 💩 but I couldn't make it get along with Vue's interpolation. I swear I used to be able to do this.

Then I tried to make this function all Typescript-y and hinted, but it just threw errors. PHPStorm has red squiggles all over it about implicit types but it works.

Reply:
This does work, but I wouldn't call it optimal or the "Vue" way 😆 

I made changes and will leave this unmerged so you can check them out, but here's what I changed:
1. The types needed to be adjusted. You had added a new property `album` to `CatalogTrack` using the type `Album`, however that type refers to a Jam album (the album returned as part of active jam data). This is really a new `CatalogAlbum` model, and I've created that and referenced it in `CatalogTrack`.
I also realized we had a bunch of `Date` types, which is really a Date object, so those needed to be changed to `strings` to be more accurate.
2. There are some cases where `v-html` is unavoidable, but this isn't one of them. In this case, the best approach is to loop over the artists using `v-for` within the template, and create your html in there. This is much more flexible than building an html string to parse.
Also added some underlines to links so it's more obvious to users when they hover.